bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/PairVLQ_x53x53_tWtW_narrow/PairVLQ_x53x53_tWtW_narrow_RH_M1000/PairVLQ_x53x53_tWtW_narrow_RH_M1000_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model VLQ_v4_4FNS_UFO  
+define ww = w+ w-
+define all = g a ve vm vt ve~ vm~ vt~ u c t d s b u~ c~ t~ d~ s~ b~ e- mu- ta- e+ mu+ ta+
+generate p p > x x~ , (x > t w+), (x~ > t~ w-)
```

Comment:
same comment here, the width needs to be set as "auto". and the decay of top and w boson will be handled by pythia?

Reply:
We are simulating in the NWA so the decay width of x is set to a nominal value in the UFO